langchain4j-vearch/src/test/java/dev/langchain4j/store/embedding/vearch/VearchContainer.java

```diff
@@ -1,16 +1,14 @@
 package dev.langchain4j.store.embedding.vearch;
 
 import org.testcontainers.containers.GenericContainer;
-import org.testcontainers.containers.wait.strategy.Wait;
 import org.testcontainers.utility.MountableFile;
 
 public class VearchContainer extends GenericContainer<VearchContainer> {
 
     public VearchContainer() {
-        super("vearch/vearch:3.4.1");
+        super("vearch/vearch:latest");
```

Comment:
I would use a fixed version instead. In the past it was hard to find the right version that was working with langchain4j.

Reply:
Good point. Will change it soon.

Reply:
What about testing agains the latest version (to make sure we are up to date) and specify supported version(s) in the [documentation](https://docs.langchain4j.dev/integrations/embedding-stores/vearch)?

Reply:
Sounds great, I'm wondering if there is a way to start containers one by one and repeat the execution of the tests, thus ensuring that our tests are passing on multiple versions.

Reply:
@katcharov already commented about it [here](https://discord.com/channels/1156626270772269217/1301238304536264818/1302014566770671678)